### PR TITLE
[DNS] Added troubleshooting info for initial domain activation

### PR DESCRIPTION
**Pull request by kodster28**
Addresses PCX-3585 and a few other bugs in the docs

### CODE REVIEW

File: content/dns/_partials/_ssltls-subdomains.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+If your main domain is using Cloudflare's [Universal SSL certificate](/ssl/edge-certificates/universal-ssl/), that certificate also covers all first-level subdomains (`blog.example.com`).
```
Comment (kodster28) [inline content/dns/_partials/_ssltls-subdomains.md:8]: Moved existing content into a partial

File: content/dns/_partials/_subdomain-customization.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (kodster28) [inline content/dns/_partials/_subdomain-customization.md:1]: Moved existing content into a partial

File: content/dns/zone-setups/troubleshooting/dns-probe-finished-nxdomain.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Though visitors sometimes encounter this error — or similarly worded messages from Safari, Edge, or Firefox — because of network or local DNS issues, it might point to an issue with your DNS records in Cloudflare.
+
+## Potential solutions
+
+If you experience `DNS_PROBE_FINISHED_NXDOMAIN` errors with a newly activated domain, review your DNS settings in the Cloudflare dashboard.
+
+Check your expected root domain (`example.com`) and any active subdomains (`www.example.com` or `blog.example.com`). If they do not resolve correctly, you may need to add a [root domain record](/dns/manage-dns-records/how-to/create-root-domain/) or a [subdomain record](/dns/manage-dns-records/how-to/create-subdomain/) in Cloudflare DNS.
+
+If you have the correct records set up, make sure those records are also  pointing to the correct origin IP address.
```
Comment (dcpena) [inline content/dns/zone-setups/troubleshooting/dns-probe-finished-nxdomain.md]:
```suggestion
If you have the correct records set up, make sure those records are also pointing to the correct origin IP address.
```

**Review by dcpena — APPROVED** [review]
One small spacing fix but otherwise 👍

**Pull request merged**